gprofiler/main.py

```diff
@@ -133,6 +137,7 @@ def __init__(
             self._system_metrics_monitor: SystemMetricsMonitorBase = SystemMetricsMonitor(self._stop_event)
         else:
             self._system_metrics_monitor = NoopSystemMetricsMonitor()
+        self._kernel_message_publisher = KernelMessagePublisher(DevKmsgReader())
```

Comment:
Make sure to handle old kernel versions which do not support `/dev/kmsg`

Reply:
Added.